tests/Integration/TransformationSystem/Test_Client_Transformation.py

```diff
@@ -182,15 +182,67 @@ def test_mix( self ):
     self.assert_( res['OK'] )
     for f in res['Value']:
       self.assertEqual( f['Status'], 'Unused' )
+      self.assertEqual( f['ErrorCount'], 0 )
     res = self.transClient.setFileStatusForTransformation( transID, 'Assigned', lfns )
     self.assert_( res['OK'] )
     res = self.transClient.getTransformationFiles( {'TransformationID':transID, 'LFN': lfns} )
     for f in res['Value']:
       self.assertEqual( f['Status'], 'Assigned' )
+      self.assertEqual( f['ErrorCount'], 0 )
     res = self.transClient.getTransformationStats( transID )
     self.assert_( res['OK'] )
     self.assertEqual( res['Value'], {'Assigned': 4L, 'Total': 4L} )
+    # Setting files MaxReset from Assigned should increment ErrorCount
+    res = self.transClient.setFileStatusForTransformation( transID, 'MaxReset', lfns )
+    res = self.transClient.getTransformationFiles( {'TransformationID':transID, 'LFN': lfns} )
+    self.assert_( res['OK'] )
```

Comment:
`assert_` is deprecated, please replace `self.assert_(res['OK'])` with `self.assertTrue(res['OK'])`